tests/console/validate_modify_existing_partition.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify the partition modified in modify_existing_partition.
+# Maintainer: Jonathan Rivrain <jrivrain@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_data';
+use Test::Assert ':all';
+
+sub run {
+    my $test_data  = get_test_data();
+    my $fs_type    = $test_data->{fs_type};
+    my $partition  = $test_data->{existing_partition};
+    my $mountpoint = $test_data->{mount_point};
+    my $part_size  = $test_data->{part_size};
```

Comment:
you can use those variables directly in your test (even interpolate them into strings when you editor does not display very good colors like in my case). Anyway it is also clear like it is, just take into account that in Perl programmers try to avoid too many declarations.